funcx_endpoint/funcx_endpoint/endpoint/endpoint_manager.py

```diff
@@ -144,6 +145,19 @@ def start_endpoint(self, name, endpoint_uuid, endpoint_config):
             print('3. Start the endpoint.')
             return
 
+        # Error handling for loading config
+        default_address = SourceFileLoader('config', endpoint_default_config.__file__).load_module().config.funcx_service_address
```

Comment:
Why are we loading the config here? We are loading the config in `funcx_endpoint/endpoint/<start_endpoint>`.
1. We should avoid loading the config again in the wrong place
2. We should use the config that we load already over there

Reply:
The `default_address` is loaded for a comparison. If we do not need the comparison, we do not need to load it.

Reply:
I see what you are trying to do now. I think that checking the address is in general a bad idea because we might have a test deployment with a different address. We'd have to do something like a blacklist of addresses in this case, which is not super great. I'd recommend not using the address at all and instead rely on other indirect markers. Let's discuss over voice what to do here.